orca-core/src/test/groovy/com/netflix/spinnaker/orca/pipeline/util/ArtifactResolverSpec.groovy

```diff
@@ -394,22 +400,66 @@ class ArtifactResolverSpec extends Specification {
     artifacts.size == 2
     artifacts*.type == ["1", "trigger"]
   }
+
   @Unroll
-  def "should resolve expected artifacts from pipeline for #expectedArtifacts using #available and prior #prior"() {
+  def "should resolve expected artifacts from pipeline for default #defaultArtifact, available #available, and prior #prior"() {
```

Comment:
I minorly refactored this and the next test, as the enormous table in the `where` clause was getting hard to read.  I pulled the one case with multiple expected artifacts into its own test case, and simplified this test given that it only had to consider one artifact.